src/Common/Editor/Full_Site/Hooks.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace TEC\Common\Editor\Full_Site;
+
+/**
+ * Class Hooks
+ *
+ * @since   TBD
+ *
+ * @package TEC\Common\Editor\Full_Site
+ */
+class Hooks extends \tad_DI52_ServiceProvider {
```

Comment:
The `register` method of this provider is empty: is this required?

Reply:
Another one that will be used in the next iteration.